File: Easy/1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n=len(nums)

        c=0
        for i in range(1, n):
            if nums[i]<nums[i-1]:
                c+=1

        if nums[0]<nums[n-1]:
            c+=1

        if c>1:
            return False

        return True

File: Easy
Easy/1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution{
    public boolean check(int[] nums){
        int n=nums.length;

        int c=0;
        for(int i=1; i<n; i++){
            if(nums[i]<nums[i-1]){
                c++;
            }
        }

        if(nums[0]<nums[n-1]){
            c++;
        }

        if(c>1)
            return false;

        return true;
    }
}